pkg/controller/publishingstrategy/publishingstrategy_controller.go

```diff
@@ -160,23 +180,142 @@ func (r *ReconcilePublishingStrategy) Reconcile(request reconcile.Request) (reco
 		}
 
 		// change Alias of resource record set of external LB in public hosted zone to internal LB
-		domainName := configv1.DNSSpec{}.BaseDomain // in form of ```samn-test.j5u3.s1.devshift.org```
-
-		// In order to update DNS we need the route53 public zone name
-		// which happens to be the domainName minus the name of the cluster
-		// Since there are NO object on cluster with just clusterName,
-		// we will index the first period and parse right
-		pubDomainName := domainName[strings.Index(domainName, ".")+1:] // pubDomainName in form of ```j5u3.s1.devshift.org```
-		apiDNSName := "api." + domainName + "."
+
 		comment := "Update api.<clusterName> alias to internal NLB"
 
+		// print out what we're passing into the UpsertARecord func
+		log.Info(fmt.Sprintf("publicDomainName is: %s", pubDomainName))
+		log.Info(fmt.Sprintf("intDNSName is: %s", intDNSName))
+		log.Info(fmt.Sprintf("intHostedZoneID is: %s", intHostedZoneID))
+		log.Info(fmt.Sprintf("apiDNSName is %s", apiDNSName))
+		log.Info(fmt.Sprintf("comment is: %s", comment))
+
 		// upsert resource record to change api.<clusterName> from external NLB to internal NLB
-		err = awsClient.UpsertARecord(pubDomainName, intDNSName, intHostedZoneID, apiDNSName, comment, false)
+		err = awsClient.UpsertARecord(pubDomainName+".", intDNSName, intHostedZoneID, apiDNSName, comment, false)
 		if err != nil {
 			log.Error(err, "Error updating api.<clusterName> alias to internal NLB")
+			return reconcile.Result{}, err
 		}
+		log.Info(comment + " successful")
 		return reconcile.Result{}, nil
 	}
 
+	// if CR is wanted the default server API to be internet-facing, we
+	// create the external NLB for port 6443/TCP and add api.<cluster-name> DNS record to point to external NLB
+	if instance.Spec.DefaultAPIServerIngress.Listening == cloudingressv1alpha1.External {
+		// get a list of all non-classic ELBs
```

Comment:
code commented out is for making a new targetGroup and listener etc. Since we have changed some of our logic this may not be needed, but we MIGHT need it later on. I would prefer to keep them around in case we need them later

Reply:
Is this supposed to be a WIP PR? Are we going to enable that part in a follow-up PR?